data.sql

```diff
@@ -126,3 +126,17 @@ VALUES
 ((SELECT id FROM animals WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-08-03'),
 ((SELECT id FROM animals WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-05-24'),
 ((SELECT id FROM animals WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2021-01-11');
+
+
+--AUDIT DATABASE
+
+-- This will add 3.594.280 visits considering you have 10 animals, 4 vets, and it will use around ~87.000 timestamps (~4min approx.)
```

Comment:
- [ ] Please make sure that the execution time before adding indexes exceeds `1000ms` as per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=If%20you%20get%20Execution%20time%3A%20X%20ms%20and%20X%20%3E%3D%201000%3A%20that%20should%20be%20enough%2C%20you%20can%20continue%20to%20the%20project%20requirements.%20%2D%20If%20you%20get%20Execution%20time%3A%20X%20ms%20and%20X%20%3C%201000%3A%20please%20go%20back%20to%20point%203.%20and%20repeat%20until%20you%20get%20a%20value%20bigger%20than%201000ms.) for the first and second query.

Reply:
Thank you for the feedback 😄 . I have implemented the change and made sure the execution time exceeds 1000ms.

Reply:
**Query 1**

- Before

![Code_qC0X1KX4hE](https://user-images.githubusercontent.com/106531695/223348924-de91917e-44ec-4e60-b1d2-b945f8cf84ac.png)

- After

![Code_HtotGCjfQP](https://user-images.githubusercontent.com/106531695/223351466-8c897048-5d9e-46ad-9f81-a256adaa57cf.png)


**Query 2**
- Before

![Code_Sv0ulxtxTE](https://user-images.githubusercontent.com/106531695/223352507-f21f6b1b-43bd-437f-bc09-f9c701dd23d2.png)

- After 

![Code_72YciZ15VB](https://user-images.githubusercontent.com/106531695/223352829-317b2d77-a29f-4c7f-8b5a-cc9533cfbf64.png)